source/sass/buyers-guide/bg-main.scss

```diff
@@ -37,7 +37,7 @@ $bp-xl: #{map-get($grid-breakpoints, xl)}; // >= 1200px
 @import "../../js/buyers-guide/components/creepiness-chart/creepiness-chart";
 @import "../../js/buyers-guide/components/likelyhood-chart/likelyhood-chart";
 @import "../../js/buyers-guide/components/filter/filter";
-@import "../../js/buyers-guide/components/social-share/social-share";
+@import "../../js/components/share-button-group/share-button-group";
```

Comment:
If this component is only used for PNI, let's leave the file in `/buyers-guide` directory.

Reply:
No, I think this was just removing the old share buttons stylesheeet since it had no custom styles and adding the share buttons styles since we're using them on buyers guide now.